manifests/backup/mysqldump.pp

```diff
@@ -28,7 +30,7 @@
     ensure     => $ensure,
     user       => "${backupuser}@localhost",
     table      => '*.*',
-    privileges => [ 'SELECT', 'RELOAD', 'LOCK TABLES', 'SHOW VIEW', 'PROCESS' ],
+    privileges => [ 'SELECT', 'RELOAD', 'LOCK TABLES', 'SHOW VIEW', 'PROCESS', 'TRIGGER' ],
```

Comment:
@cyberious thinks you should add an if-statement around this to make sure TRIGGERS is only added if people want it.